pimcore/lib/Pimcore/Bundle/AdminBundle/Controller/Admin/ElementController.php

```diff
@@ -563,6 +563,28 @@ public function deleteVersionAction(Request $request)
         return $this->adminJson(['success' => true]);
     }
 
+    /**
+     * @Route("/element/delete-all-version")
```

Comment:
can we change the route to `/element/delete-all-versions` ... its only a small detail, but you know ;-) ...

Reply:
Done! :)